packages/api/src/utils/auth.js

```diff
@@ -29,6 +26,10 @@ export async function validate(event, { sentry }) {
       if (user.data) {
         const key = user.data.keys.find((k) => k?.secret === token)
         if (key) {
+          log.setUser({
+            email: user.data.email,
```

Comment:
I'm not sure we should be adding user emails to our logs...ID is enough to lookup their email. Do we need both?